tests/nnstreamer_sink/unittest_sink.cpp

```diff
@@ -670,13 +670,17 @@ _setup_pipeline (TestOption & option)
           "videoconvert ! tensor_sink name=test_sink", option.num_buffers);
       break;
     case TEST_TYPE_VIDEO_RGB_SPLIT:
-      /** video stream with tensor_split */
+      /**
+       * video stream with tensor_split
+       * This often causes test errors in heavily loaded systems. Thus, set the famerate 0/1.
+       * @todo Find a way to do it in "sync=false" mode despite of those queues.
+       */
       str_pipeline =
           g_strdup_printf
           ("videotestsrc num-buffers=%d ! videoconvert ! video/x-raw,width=160,height=120,format=RGB,framerate=(fraction)30/1 ! "
           "tensor_converter ! tensor_split name=split tensorseg=1:160:120,1:160:120,1:160:120 "
           "split.src_0 ! queue ! tensor_sink "
-          "split.src_1 ! queue ! tensor_sink name=test_sink "
+          "split.src_1 ! queue ! tensor_sink sync=true name=test_sink "
```

Comment:
set 'sync=true' actually no effect (default value is true in base-sink).

how about adding sleep time with sync=false before changing state to null?
```
  gst_element_set_state (g_test_data.pipeline, GST_STATE_PLAYING);
  g_main_loop_run (g_test_data.loop);
  g_usleep (100 * 1000);
  gst_element_set_state (g_test_data.pipeline, GST_STATE_NULL);
```

Reply:
uh... let's look at the unit-testing performance issue later. We are not experiencing the issue after a few updates in the CI system.

Reply:
I'm removing unit-test updating commit from this PR.